test/flows/check_travel_during_coronavirus_flow_test.rb

```diff
@@ -9,7 +9,7 @@ class CheckTravelDuringCoronavirusFlowTest < ActiveSupport::TestCase
   setup do
     testing_flow CheckTravelDuringCoronavirusFlow
     stub_foreign_travel_advice
-    stub_worldwide_api_has_locations(%w[spain ireland italy poland ukraine])
+    stub_worldwide_api_has_locations(%w[spain ireland italy poland ukraine russia])
```

Comment:
Um, this is still here. Why?

Reply:
Russia needs to exist here - and so do all of the countries in the `PASSENGER_LOCATOR_FORM_EXEMPT_COUNTRIES` list. This is because in the `country_phrase(countries)` method we call the WorldLocation API to get the country title for the humanised string - and that call needs to be stubbed for the tests to pass. However, if we hard-code the list for the start page (only), then we won't need to do this as we have the title in `country_locations` and can get it out of there for the user selected countries.